release/app-service-configurable.yaml

```
@@ -0,0 +1,20 @@
+---
+name: 'app-service-configurable'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+repo: 'https://github.com/edgexfoundry/app-service-configurable.git'
+commitId: '4ec47cdd8cea7a0176fb01ed614db2877bbe7fef'
```

```suggestion
commitId: 'bfff9069ec602defc1c67fa58b1ab94b3f5a4b73'
```
That is the final commit for release.